docs/src/service-workers-experimental-network-events.md

```diff
@@ -1,28 +1,27 @@
 ---
-id: service-workers
-title: "Service Workers"
+id: service-workers-experimental
```

Comment:
This guide will only be linked to from the issue. Removing from the sidebar momentarily.